src/components/home.tsx

```diff
@@ -1,67 +1,35 @@
-import React, { useState, useEffect, useReducer, forwardRef, createRef } from "react";
+import React, { useReducer, useState, forwardRef, createRef } from "react";
 import { Grid } from "semantic-ui-react";
 
-import { Image, defaultImages, ImagesContext } from "./images";
+import { defaultImages, ImagesContext } from "./images";
 import { Search, SearchType } from "./search";
 import { ConfigMenu } from "./menu";
 import { Chart } from "./chart";
-import { getAlbum, getGame, getMovie, getSeries } from "./fetcher";
+import { Image } from "./images";
 import { onResults } from "./results";
 import { ConfigContext, ConfigInitialState } from "./config";
 import { configReducer, imagesReducer } from "./reducers";
 
 export const Home: React.FC = () => {
-  const [resultsImgs, setResultsImgs] = useState<Image[]>([]);
-  // TODO: move to the component / confg?
-  const [search, setSearch] = useState("");
-  const [searchType, setSearchType] = useState(SearchType.Music);
   const [config, dispatchConfig] = useReducer(configReducer, ConfigInitialState);
   const [images, dispatchImages] = useReducer(imagesReducer, defaultImages(10, 10));
-
+  const [search, setSearch] = useState({
+    search: "",
+    searchType: SearchType.Music,
+    resultsImgs: Array<Image>(),
+  });
```

Comment:
This should be split up, since any change on this state will trigger not only a re-render but also any useEffects that are set up.